protocols/direct-funding.go

```diff
@@ -295,15 +298,16 @@ func (s DirectFundingObjectiveState) applySignature(signature state.Signature, t
 	index, ok := s.ParticipantIndex[signer]
 	if !ok {
 		return fmt.Errorf("signature recieved from unrecognized participant 0x%x", signer)
-	}
+	} else {
 
-	if turnNum == 0 {
-		s.PreFundSigned[index] = true
-	} else if turnNum == 1 {
-		s.PostFundSigned[index] = true
-	}
+		if turnNum == 0 {
+			s.PreFundSigned[index] = true
+		} else if turnNum == 1 {
+			s.PostFundSigned[index] = true
+		}
```

Comment:
```
if !ok {
 return err
}

//regular flow
```
vs
```
if !ok {
 return err
} else {
  // regular flow
}
```

The "implicit else" style of error-handling and control flow is not generally my favorite, but it seems ubiquitous in go. This is probably because the (comma, ok) and `func() (returnType, error)` idioms are so common that the alternative to implicit-else style is for everything to move one-indent-to-the-right every time something doesn't fail.

Reply:
4c0506a